packages/ses/src/make-hardener.js

```diff
@@ -218,21 +220,53 @@ export const makeHardener = () => {
             enqueue(desc.set, `${pathname}(set)`);
           }
         });
-      }
+      };
+
+      const freezeAndTraverse =
+        FERAL_STACK_GETTER === undefined
+          ? // On platforms without v8's error own stack accessor problem,
+            // don't pay for any extra overhead.
+            baseFreezeAndTraverse
+          : obj => {
+              if (isError(obj)) {
+                // Only pay the overhead if it first passes this cheap isError
+                // check. Otherwise, it will be unrepaired, but won't be judged
+                // to be a passable error anyway, so will not be unsafe.
+                const stackDesc = getOwnPropertyDescriptor(obj, 'stack');
+                if (
+                  stackDesc &&
+                  stackDesc.get === FERAL_STACK_GETTER &&
```

Comment:
Test for either the getter or setter?
```suggestion
                  (stackDesc.get === FERAL_STACK_GETTER || stackDesc.set === FERAL_STACK_SETTER) &&
```

Reply:
Done, but with `&&` rather than `||` to be more conservative. But either they are both set or neither is set, so cannot actually make a difference.

Resolving.

Reply:
I believe I didn't express myself correctly. I was trying to handle the case where an error object would show up with either a getter or a setter own prop. Arguably someone would have to go muck with the prop descriptor before hardening the error object, so probably not worth bothering about.

Reply:
This test here is only testing whether we're on a platform suffering from v8's misbehavior. 

Ok to reresolve?

Reply:
My understanding is that we already know we're on an affected platform, but here we're testing whether the error object we found exhibit the bad behavior. It's possible the error object got modified already (e.g. stack prop removed, or replaced with data). The question is which kind of modifications we want to handle here (in particular whether we want to handle a setter only stack prop if we find one)